### PR TITLE
Update usage infos for Admin API commands with admin flags

### PR DESCRIPTION
**Pull request by dopey**

### CODE REVIEW

File: command/ca/provisionerbeta/update.go [hunk 1]
```diff
@@ -69,7 +79,9 @@ IID (AWS/GCP/Azure)
 [**--azure-tenant**=<id>] [**--azure-resource-group**=<name>]
 [**--instance-age**=<duration>] [**--iid-roots**=<file>]
 [**--disable-custom-sans**] [**--disable-trust-on-first-use**]
-[**--ca-url**=<uri>] [**--root**=<file>]`,
+[**--admin-cert=<file>] [**--admin-key=<file>]
+[**--admin-provisioner**=<string>] [**--admin-subject=<string>]
+[**--password-file**=<file>] [**--ca-url**=<uri>] [**--root**=<file>]`,
```
Comment (maraino) [inline command/ca/provisionerbeta/update.go:84]: Missing `**` in all the examples.

**Review by maraino — CHANGES_REQUESTED** [review]
There are missing `**` in most admin flags, and I've suggested a change in the usage of a flag.

File: command/ca/admin/update.go [hunk 2]
```diff
@@ -18,16 +18,18 @@ func updateCommand() cli.Command {
...
+[**--password-file**=<file>] [**--ca-url**=<uri>] [**--root**=<file>]`,
 		Flags: []cli.Flag{
 			cli.BoolFlag{
 				Name:  "super",
 				Usage: `Update the admin with super-admin privileges.`,
 			},
 			cli.StringFlag{
 				Name:  "provisioner",
-				Usage: `Filter admin by provisioner name`,
+				Usage: `The <string> name of a provisioner by which to filter admins.`,
```
[the remark this answers is not in the document]
Comment (dopey) [reply command/ca/admin/update.go:32]: I changed the UsageText above to use `string` because that matches the other args.

**Review by maraino — APPROVED** [review]
lgtm

**Pull request merged**